RecoHI/Configuration/python/Reconstruction_hiSignal_cff.py

```diff
@@ -11,8 +11,9 @@
 
 #Tracks
 hiSignalGlobalPrimTracks = hiGlobalPrimTracks.clone()
-hiSignalSelectedTracks = hiSelectedTracks.clone()
-hiSelectedTracks.src = cms.InputTag("hiSignalGlobalPrimTracks")
+hiSignalSelectedTracks = hiSelectedTracks.clone(
```

Comment:
Here the `hiSignalSelectedTracks` are updated with the input collection "hiSignalGlobalPrimTracks", but then later the `heavyIonTrackingTask` is feeed with the (unmodified) `hiSelectedTracks`: I wonder how it can even work
I suppose you already tested it, e.g. with the HI workflows in the matrix tests (140.43, 140.46, 158.01): but it is possible that you wouldn't get an error in that case, because it does not seem to me that this Reconstruction_hiSignal_cff.py is ever included in any actual reconstruction sequence.

Let then involve the HI reco contact, @mandrenguyen : 
- I already noticed in [my comment](https://github.com/cms-sw/cmssw/pull/32171/files#r526776262) for the previous version of this PR that already the original formulation of this PR was bugged; I don't think this update is actually fixing it
- I don't see this Reconstruction_hiSignal_cff.py ever used in the configs in CMSSW
- Do you have any suggestion (based on a possible usage of this python snippet) for fixing it? If this file isn't really used anywhere, wouldn't it be more convenient to profit of this PR in order to completely remove it, instead of spending more efforts in trying to fix it?